src/ducks/components/PushBanners/helpers.spec.js

```diff
@@ -0,0 +1,104 @@
+import { makePushBanner } from './helpers'
+
+jest.mock('./BannerForAA', () => 'BannerForAA')
+jest.mock('./BannerForPass', () => 'BannerForPass')
+
+describe('makePushBanner', () => {
+  describe('should return the banner for AA', () => {
+    it('when no oAuth client installed for AA and Pass mobile and switches not set in database', () => {
```

Comment:
nit: AA is an internal code name, we should talk about Flagship App instead
(also it is better to use `nor` instead of `and` here)

```suggestion
    it('when no oAuth client installed for Flagship App nor Pass mobile and switches not set in database', () => {
```

Reply:
Ah j'avais pas vu que y avait autant de ref à "AA", merci pour la modif 👍